locales/en.default.schema.json

```
@@ -146,6 +146,24 @@
         }
       }
     },
+    "search_input": {
+      "name": "Search input",
+      "settings": {
+        "header": {
+          "content": "Product suggestions"
+        },
+        "predictive_search_enabled": {
+          "label": "Enable product suggestions",
+          "info": "Affects the search bars in the header and on the search results page."
+        },
+        "predictive_search_show_vendor": {
+          "label": "Show vendor" 
+        },
+        "predictive_search_show_price": {
+          "label": "Show price"
```

```suggestion
          "label": "Show price",
          "info": "Visible when product suggestions enabled."
```
Based on @gretchen-willenberg suggestion!

---

Should there be an "are" before enabled? 

> Visible when product suggestions are enabled

---

No, please. `Enabled` as past-tense verb is okay; if we add `are` we get passive voice, which is generally avoided except for some error messaging. Thank you!

---

The period is necessary because it is an info text under the setting. [Example here](https://screenshot.click/21-08-ft4zl-woww5.png)